index.html

```diff
@@ -14,10 +14,19 @@
          
     </nav>
 
+    <nav class="dsk-header-logo">
+        <a href="#"><img src="images/My Logo.svg" alt="" class="logo-2"></a>
+        <ul class="nav-content">
+            <li><img src="images/Portoflio.svg" alt=""></li>
+            <li><img src="images/About.svg" alt=""></li>
+            <li><img src="images/Contact.svg" alt=""></li>
+        </ul>
+    </nav>
+
```

Comment:
- [ ] Nice work so far 👏 . However, You're meant to keep the mobile design perfect as before, to make the whole app adjustable on a desktop view. you can see from the image below, the extra `header` for navigation has clustered the mobile design. Kindly fix this to give a better user experience and make your wor professional 👍 

## View:
![image](https://user-images.githubusercontent.com/50990925/222939977-ea4650e2-dbae-4575-874f-4d34de323ec5.png)